radio/src/keys.cpp

```diff
@@ -469,7 +469,7 @@ bool keysPollingCycle()
       evt |= i;
 
       // Radio with single PAGEDN key
-      if (keysGetSupported() & (1 << KEY_PAGEUP)) {
+      if (!(keysGetSupported() & (1 << KEY_PAGEUP))) {
```

Comment:
Thx for fixing this! We should probably have a dedicated inline function to avoid this mess.